diff --git a/third_party/remote_config/common.bzl b/third_party/remote_config/common.bzl
index 2d627e26cb..aaca8cafc9 100644
--- a/third_party/remote_config/common.bzl
+++ b/third_party/remote_config/common.bzl
@@ -265,7 +265,7 @@ def realpath(repository_ctx, path, bash_bin = None):
     if bash_bin == None:
         bash_bin = get_bash_bin(repository_ctx)
 
-    return execute(repository_ctx, [bash_bin, "-c", "realpath \"%s\"" % path]).stdout.strip()
+    return execute(repository_ctx, [bash_bin, "-c", "readlink -f \"%s\"" % path]).stdout.strip()
 
 def err_out(result):
     """Returns stderr if set, else stdout.
