diff --git a/third_party/gpus/cuda_configure.bzl b/third_party/gpus/cuda_configure.bzl
index af1bc96f00..2fd0bef286 100644
--- a/third_party/gpus/cuda_configure.bzl
+++ b/third_party/gpus/cuda_configure.bzl
@@ -373,8 +373,8 @@ def _cuda_include_path(repository_ctx, cuda_config):
             )
     inc_entries = []
     if target_dir != "":
-        inc_entries.append(target_dir)
-    inc_entries.append(cuda_config.cuda_toolkit_path + "/include")
+        inc_entries.append(repository_ctx.path(target_dir).realpath)
+    inc_entries.append(repository_ctx.path(cuda_config.cuda_toolkit_path + "/include").realpath)
     return inc_entries
 
 def enable_cuda(repository_ctx):
