diff --git a/third_party/gpus/cuda_configure.bzl b/third_party/gpus/cuda_configure.bzl
index af1bc96f00..2fd0bef286 100644
--- a/third_party/gpus/cuda_configure.bzl
+++ b/third_party/gpus/cuda_configure.bzl
@@ -373,8 +373,8 @@ def _cuda_include_path(repository_ctx, cuda_config):
             )
     inc_entries = []
     if target_dir != "":
-        inc_entries.append(target_dir)
-    inc_entries.append(cuda_config.cuda_toolkit_path + "/include")
+        inc_entries.append(repository_ctx.path(target_dir).realpath)
+    inc_entries.append(repository_ctx.path(cuda_config.cuda_toolkit_path + "/include").realpath)
     return inc_entries
 
 def enable_cuda(repository_ctx):
diff --git a/third_party/gpus/find_cuda_config.py b/third_party/gpus/find_cuda_config.py
index 39f2c21d3f..c41e9d551b 100644
--- a/third_party/gpus/find_cuda_config.py
+++ b/third_party/gpus/find_cuda_config.py
@@ -202,7 +202,7 @@ def _not_found_error(base_paths, relative_paths, filepattern):
 def _find_file(base_paths, relative_paths, filepattern):
   for path in _cartesian_product(base_paths, relative_paths):
     for file in glob.glob(os.path.join(path, filepattern)):
-      return file
+      return os.path.realpath(file)
   raise _not_found_error(base_paths, relative_paths, filepattern)
 
 
