diff --git a/third_party/gpus/cuda_configure.bzl b/third_party/gpus/cuda_configure.bzl
index af1bc96f00..2da4e4fc74 100644
--- a/third_party/gpus/cuda_configure.bzl
+++ b/third_party/gpus/cuda_configure.bzl
@@ -366,10 +366,7 @@ def _cuda_include_path(repository_ctx, cuda_config):
     for one_line in result.stderr.splitlines():
         if one_line.startswith("#$ _TARGET_DIR_="):
             target_dir = (
-                cuda_config.cuda_toolkit_path + "/" + one_line.replace(
-                    "#$ _TARGET_DIR_=",
-                    "",
-                ) + "/include"
+                cuda_config.cuda_toolkit_path + "/include"
             )
     inc_entries = []
     if target_dir != "":
