diff --git a/third_party/gpus/find_cuda_config.py b/third_party/gpus/find_cuda_config.py
index 39f2c21d3f..e88409df64 100644
--- a/third_party/gpus/find_cuda_config.py
+++ b/third_party/gpus/find_cuda_config.py
@@ -171,6 +171,7 @@ def _header_paths():
   """Returns hard-coded set of relative paths to look for header files."""
   return [
       "",
+      "targets/x86_64-linux/include",
       "include",
       "include/cuda",
       "include/*-linux-gnu",
